pkgs/servers/computing/slurm/default.nix

```diff
@@ -49,15 +52,17 @@ stdenv.mkDerivation rec {
     curl python3 munge perl pam zlib
       libmysqlclient ncurses gtk2 lz4 rdma-core
       lua hwloc numactl readline freeipmi shadow.su
-      pmix
+      pmix json_c libjwt libyaml
   ] ++ lib.optionals enableX11 [ xorg.xauth ];
 
   configureFlags = with lib;
     [ "--with-freeipmi=${freeipmi}"
       "--with-hwloc=${hwloc.dev}"
+      "--with-json=${json_c.dev}"
+      "--with-jwt=${libjwt}"
       "--with-lz4=${lz4.dev}"
       "--with-munge=${munge}"
-      "--with-zlib=${zlib}"
```

Comment:
Is it still built with zlib properly?

Reply:
I don't think slurm is still using zlib.
There is no "--with-zlib" configure option.
grep doesn't show any potential uses of zlib.
The NEWS file indicates that zlib was replaced with lz4 in 16.05.0rc1.

Reply:
ah, okay. Please remove it also from the inputs.